src/main/java/net/gleske/scmfilter/impl/trait/RegexSCMHeadFilterTrait.java

```diff
@@ -49,8 +49,6 @@
 /**
  * Decorates a {@link SCMSource} with a {@link SCMHeadPrefilter} that excludes {@link SCMHead} instances with names that
  * do not match a user supplied regular expression.
- *
- * @since 2.2.0
```

Comment:
What is the purpose of this annotation?  This class was originally copied from CloudBees released code and updated for this plugin.  Perhaps @stephenc can help answer?

Reply:
The convention in Jenkins is to put `@since` tags on any new things added since the `@since` on the class.

(On phone so I lack full context)

AIUI this is a new plugin so you’d just have `@since 1.0` (or similar) on the class and no tags on the methods or fields

Reply:
Ah okay, at this time I’m not concerned with its removal.  Just wanted to understand a little better.